tmt/plugins/__init__.py

```diff
@@ -14,11 +15,11 @@
 log = fmf.utils.Logging('tmt').logger
 
 
-def explore():
+def explore() -> None:
     """ Explore all available plugins """
 
     # Check all tmt steps for native plugins
-    root = os.path.dirname(os.path.realpath(tmt.__file__))
+    root: str = os.path.dirname(os.path.realpath(tmt.__file__))
```

Comment:
What was the error message that made you add an explicit `str` annotation? I would expect the type of `root` to be easy to infer, thanks to `os.path.dirname` being a standard library function with a known return type value (`AnyStr`, or `str` in Python 3).

Reply:
No errors, I just overtyped it a little :) Reversing the lone, thanks